src/backends/github/implementation.js

```diff
@@ -138,7 +138,7 @@ export default class GitHub {
       const promises = [];
       branches.map((branch) => {
         promises.push(new Promise((resolve, reject) => {
-          const slug = branch.ref.split("refs/heads/cms/").pop();
+          const slug = branch.ref.split(`refs/heads/${this.api.cms_branch_prefix}`).pop();
```

Comment:
This will break sites that don't use master and have existing editorial workflow entries. This kind of backwards compatibility bug is the primary concern with a change like this. We almost need a way to find and transition existing entries, or by some other means ensure that that the change is transparent for existing entries.

Reply:
@erquhart Doesn't line 14 in API.js address this concern?

```js
this.cms_branch_prefix = this.branch === 'master' ? 'cms/' : `cms-${this.branch}/`;
```